### DISCUSSION TITLE
Make it optional, please 🙏

### DISCUSSION THREAD

**Discussion by diazweb** [community/community#15461] [Other Feature Feedback, Questions, & Ideas]
The **"For You"** tab should be optional and the **"Following"** tab too. I am not a fan of having more distractions, would be great to have an option to just show my own repos and that's it. There is no need to invest time in stuff that we don't need, please don't make this another social media crap, it's fine like it was at the beginning.

Thanks

**Comment by pavelloz** [comment]
I agree. Especially when it shows things because you "collaborated with someone" ... 6 years ago.

**Reply by Jxpro** [reply]
Could you please tell me how you know that you collaborated with someone 6 years ago?

Because there is a person on my "For You" tab too, but I didn't follow this person, and I didn't star his project, and I didn't even remember ever cooperating with him.

How can I know what contact I've had with the people who appear in my "For You" tab?

**Reply by pavelloz** [reply]
I clicked three dots next to the comment and it said that.

Example:
![image](https://user-images.githubusercontent.com/546845/165850002-2524041b-4def-4350-8b57-2f6f4485cc12.png)

**Reply by Jxpro** [reply]
I am very grateful for your reply. That's a really good idea.

Also I would like to know where can I get more details, such as time, specific projects of collaboration. 

Because I can't recall the time I collaborated with him.

![image](https://user-images.githubusercontent.com/87376573/165862799-72c3eee9-9d8c-4d65-a52a-a467300618dc.png)

**Reply by paulb-peterson** [reply]
How do we suppress the message, "The Git repository appears to be owned by another user on your machine?"

**Comment by fjeddy** [comment]
It's Microsoft, they touch it and then boom....

Soon enough they are going to force you to use a Microsoft account for better tracking, oh wait, sorry "For improved security". lol.
